src/app/core/service/application-instance-management.service.ts

```diff
@@ -75,7 +76,7 @@ export class ApplicationInstanceManagementService {
             })
         };
         // TODO we have to create a proper model for interface ressources delivered by the backend
-        return this.http.get<{ interfaces: Array<any> }>(serviceTemplateInstance._links['boundarydefinitions/interfaces'].href, httpOptions)
+        return this.http.get<InterfaceList>(serviceTemplateInstance._links['boundarydefinitions/interfaces'].href, httpOptions)
```

Comment:
Does this fix the todo in line 78? If so, delete todo.

Reply:
Unfortunately not. The container still creates a map of operations instead of a list.
The comment refers to lines 83-94.